Extension/src/LanguageServer/cppBuildTaskProvider.ts

```diff
@@ -366,13 +370,13 @@ class CustomBuildTaskTerminal implements Pseudoterminal {
                     if (_error) {
                         telemetry.logLanguageServerEvent("cppBuildTaskError");
                         const dot: string = (stdout || _stderr) ? ":" : ".";
-                        this.writeEmitter.fire(`Build finished with error${dot}${this.endOfLine}`);
+                        this.writeEmitter.fire(localize("build_finished_with_error", "Build finished with error{0}{1}", dot, this.endOfLine));
```

Comment:
Similar feedback as above.  Avoid including vars in calls to localize() unless they are significant to how the line is translated.  Alternatively, you could use the other prototype for localize(), which allows a 'comment' field to be passed in, in which you can provide guidance to the loc team about how to translate the string.